### PR TITLE
added mail templates for govm

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6536
https://github.com/bcgov/entity/issues/6536

*Description of changes:*

> new invitation templates added

> two new org status added

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/email_templates/govm_business_invitation_email.html [hunk 1]
```diff
@@ -52,14 +52,15 @@
                 <tr>
                     <td>
                         <h1 style="margin: 0; line-height: 34px; letter-spacing: -1px; font-family: Verdana, Arial, sans-serif; font-size: 24px; font-weight: 700;">
-                            Your BC Registries account <strong>{{ org_name }}</strong> has been created.
+                            You've been invited to create BC Registries account.
                         </h1>
                         <p style="margin-top: 25px; margin-bottom: 0; line-height: 24px; font-family: Verdana, Arial, sans-serif; font-size: 16px;">
-                            BC Registries staff have approved your organization's account to search for minstry accounts.<br/> To complete the account setup, click the link below and create your Username and Password. Then login to your account and manage your team's access to Director Search.  </p>
+                            BC Registries staff have invited you to create an Business Registry account. You are the account administrator.<br/> To complete the account setup, click the button below and create your account. Then login to your account and manage your team members.
```
Comment (sumesh-aot) [inline auth-api/email_templates/govm_business_invitation_email.html:58]: I think;
`an Business Registry account`
should be
`a Business Registry account`

File: auth-api/email_templates/govm_member_invitation_email.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
```
Comment (sumesh-aot) [inline auth-api/email_templates/govm_member_invitation_email.html:1]: Do we need a separate template for this? Can't we use the one existing?

File: auth-api/migrations/versions/4d47b5569629_govm_org_status_added.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                             column('code', String),
+                             column('description', String),
+                             column('default', Boolean)
+                             )
+    op.bulk_insert(
+        org_status_table,
+        [
+            {'code': 'PENDING_INVITE_ACCEPT', 'description': 'User invited to his account.Waiting user action',
+             'default': False},
+            {'code': 'PENDING_STAFF_REVIEW', 'description': 'Waiting for staff review for user',
```
Comment (sumesh-aot) [inline auth-api/migrations/versions/4d47b5569629_govm_org_status_added.py:32]: Can we use PENDING_STAFF_REVIEW for affidavit review as well? Lot of statuses will complicate reporting and queries

**Review by sumesh-aot — APPROVED** [review]
Please check comments

File: auth-api/migrations/versions/4d47b5569629_govm_org_status_added.py (hunk 3 above)
Comment (saravanpa-aot) [reply auth-api/migrations/versions/4d47b5569629_govm_org_status_added.py:32]: let me see...

File: auth-api/email_templates/govm_member_invitation_email.html (hunk 2 above)
Comment (saravanpa-aot) [reply auth-api/email_templates/govm_member_invitation_email.html:1]: the IDIR username and password wording is different...

File: auth-api/email_templates/govm_business_invitation_email.html (hunk 1 above)
Comment (saravanpa-aot) [reply auth-api/email_templates/govm_business_invitation_email.html:58]: will fix it.thanks

**Pull request merged**